src/main/java/de/tum/in/www1/artemis/service/StatisticsService.java

```diff
@@ -0,0 +1,95 @@
+package de.tum.in.www1.artemis.service;
+
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.stereotype.Service;
+
+import de.tum.in.www1.artemis.repository.StatisticsRepository;
+
+@Service
+public class StatisticsService {
+
+    private final StatisticsRepository statisticsRepository;
+
+    public StatisticsService(StatisticsRepository statisticsRepository) {
+        this.statisticsRepository = statisticsRepository;
+    }
+
+    public Integer getLoggedInUsers(Long span) {
```

Comment:
The parameter name span confuses me here. It is not the span you defined in the enum? It looks to be just a number of days

Reply:
Yes, sorry, like mentioned in the PR description, this is deprecated code which will be refactored after this PR and ist just here for semantic purposes.